modules/administration/pages/troubleshooting/tshoot-oes-repos-enablement.adoc

```diff
@@ -0,0 +1,30 @@
+[[troubleshooting-oes-repositories-enablement]]
+= Troubleshooting OES repositories enablement
+
+To enable Open Enterprise Server (OES) on {productname} Server, follow the procedure described.
+ 
+.Procedure: Enabling OES repositories
+[role=procedure]
+
+. Ensure you have mirror credentials from Microfocus with access to OES.
++
+. Log in to {productname} Server.
++
+. Go to [guimenu]``Admin > Setup Wizard > Organization Credentials``.
++
+. Make sure you already have SUSE credentials for {productname}.
++
+. Select option for adding new and type in Microfocus credentials.
++
+. Go to [guimenu]``Admin > Setup Wizard > Organization Credentials`` and wait for the refresh operation to complete.
++
+. OES should be visible on the refreshed product list. 
+  You can now enable it like any other products.
+
+
+[IMPORTANT]
+====
+Do not change [path]``additional_products.json`` or [path]``java.product_tree_tag``.
+====
```

Comment:
@0rnela can you remove this? No customer knows about it. It was just our supporters who scanned bugzilla and found comments about it as the OES Development and QA Team need to use it for beta testing when they want to see it before we change it to "released". But let's not confuse the customer with it.